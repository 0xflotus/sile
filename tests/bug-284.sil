\begin[papersize=a6]{document}
\font[family=Libertinus Serif,size=11pt,language=tr]
\set[parameter=document.lskip,value=15pt]
\set[parameter=document.rskip,value=10pt]
\noindent
Çünkü Müjde’den utanmıyorum. Müjde iman eden herkesin –önce Yahudiler’in, sonra Yahudi olmayanların– kurtuluşu için Tanrı gücüdür.
\end{document}