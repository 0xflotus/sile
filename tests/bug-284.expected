<tests/bug-284.sil>
Set paper size 	419.5275636	595.275597
Begin page
Mx 	35.9764
My 	41.8732
Set font 	Gentium;16;400;;normal;;LTR
T	126 174 78 75 174	(Çünkü)
Mx 	81.3894
T	45 174 74 68 69 193 68 69 78	(Müjde’den)
Mx 	153.8727
T	85 84 65 78 77 176 89 79 82 85 77	(utanmıyorum)
Mx 	244.8180
T	14	(.)
Mx 	251.5358
T	45 174 74 68 69	(Müjde)
Mx 	295.8160
T	73 77 65 78	(iman)
Mx 	332.2290
T	69 68 69 78	(eden)
Mx 	367.2123
T	72 69 82	(her)
Mx 	389.7670
T	13	(-)
Mx 	35.9764
My 	61.0732
T	75 69	(ke)
Mx 	51.3670
T	83 73 78	(sin)
Mx 	73.7644
T	190	(–)
Mx 	81.2722
T	168 78 67 69	(önce)
Mx 	115.5837
T	57 65 72 85 68 73 76 69 82 193 73 78	(Yahudiler’in)
Mx 	197.2868
T	12	(,)
Mx 	204.0045
T	83 79 78 82 65	(sonra)
Mx 	243.7925
T	57 65 72 85 68 73	(Yahudi)
Mx 	293.6587
T	79 76 77 65 89 65 78 76 65 82 176 78 190	(olmayanların–)
Mx 	35.9764
My 	80.2732
T	75 85 82 84 85 76 85 318 85	(kurtuluşu)
Mx 	104.4978
T	73 154 73 78	(için)
Mx 	133.2457
T	52 65 78 82 176	(Tanrı)
Mx 	173.1577
T	71 174 67 174 68 174 82	(gücüdür)
Mx 	228.0405
T	14	(.)
[1] Mx 	207.4176
My 	553.7327
Set font 	Gentium;10;400;;normal;;LTR
T	17	(1)
End page
Finish

